main.go

```diff
@@ -124,7 +124,7 @@ func main() {
 			panic(fmt.Sprintf("`%s` unknown command", os.Args[1]))
 		}
 	} else {
-		//TODO (#7) implement a map for options instead of println'ing them all there
+		//TODO (#8) implement a map for options instead of println'ing them all there
```

Comment:
You were correct before the change. It should be #7. :)

Reply:
oh, i thought #7 was the CI issue lmao

Reply:
this does bring a new a new interesting issue, that TODO's do need to be kept in sync with whats reported  on github

Reply:
@fabulousduck lol my bad. The TODO should have the id of the issue it's associated with. So it should be `TODO(#9)`. We both fucked up here. :D

Reply:
@fabulousduck that's why it is better to just leave unreported TODO and let me fuck up alone :D

Reply:
haha, sure ;)